### PR TITLE
change output format

### PR DESCRIPTION
**Pull request by KSchlobohm**
## Resolves
- #267
- #268 
- #269 

## Changes impacts
- bash scripts for
   - createAppRegistration.sh
   - addLocalIPToSqlFirewall.sh
   - getSecretsForLocaldev.sh
   - makeSqlUserAccount.sh

### CODE REVIEW

**Review by kwkraus — CHANGES_REQUESTED** [review]
Having issues with the chmod command on the `createAppRegistrations.sh` file outlined in readme.md.

Need to have a chat on what needs to change in DevContainers on Windows to make this work.

File: README.md [hunk 1]
```diff
@@ -222,7 +222,8 @@ App Configuration so that the web app can read this data.
 <td>
 
 ```bash
-bash ./infra/createAppRegistrations.sh -g "$myEnvironmentName-rg"
+chmod +x createAppRegistrations.sh
```
[the remark this answers is not in the document]
Comment (KSchlobohm) [reply README.md]: unique to WSL experiences
```
sed "s/$(printf '\r')\$//" -i ./infra/createAppRegistrations.sh
```

[the remark this answers is not in the document]
Comment (KSchlobohm) [reply README.md]: re-added comment to doc to help readers find the know issue section

[the remark this answers is not in the document]
Comment (kwkraus) [reply README.md]: @KSchlobohm the `chmod +m createAppRegistrations.sh` command still fails with directory not found.  

Command that worked after running `sed` command on WSL was `chmod +m ./infra/createAppRegistrations.sh`

**Review by kwkraus — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kwkraus) [reply README.md]: this is not an issue, running now.

**Pull request merged**